src/executables/csql_result_format.c

```diff
@@ -1249,7 +1255,30 @@ string_to_string (const char *string_value, char string_delimiter, char string_i
       return NULL;
     }
 
-  if ((return_string = (char *) malloc (length + 4)) == NULL)
+  if (change_single_quote == true)
+    {
+      ptr = (char *) string_value;
+
+      while (*ptr != '\0')
+	{
+	  if (*ptr == '\'')
+	    {
+	      num_found++;
+	    }
+	  ptr++;
+	}
+    }
+
+  if (num_found == 0)
+    {
+      return_string = (char *) malloc (length + 4);
+    }
+  else
+    {
+      return_string = (char *) malloc (length + 4 + num_found);
+    }
+
+  if (!return_string)
```

Comment:
I think line# 1272 ~ 1284 could be rewrited as:
if ((return_string = (char *) malloc (length + 4 + num_found)) == NULL)

Reply:
thank you, i will change code as your recommend.